tests/ert3/console/integration/test_cli.py

```diff
@@ -121,3 +122,132 @@ def test_cli_record_load_not_existing_file(tmpdir):
     with patch.object(sys, "argv", args):
         with pytest.raises(SystemExit):
             ert3.console.main()
+
+
+def _assert_done_or_pending(captured, experiments, done_indices):
+    lines = [
+        line.strip()
+        for line in captured.out.splitlines()
+        if "have run" not in line and "can be run" not in line and line.strip()
+    ]
+
+    done = (" ").join(lines[: len(done_indices)]).split()
```

Comment:
Any reason you are not simply doing `lines[: len(done_indices)]`?

Reply:
Also, I just did some testing locally and I think `lines[:len(done_indices)]` should make both `black` and `flake8` happy 🤷 Such that we do not have to add the `E203` exception for flake8...

Reply:
> Any reason you are not simply doing `lines[: len(done_indices)]`?

The original reason was that a line may contain multiple words, i.e. the plan was to have some failed/success status. But that is indeed not necessary anymore.

Reply:
> Also, I just did some testing locally and I think `lines[:len(done_indices)]` should make both `black` and `flake8` happy 🤷 Such that we do not have to add the `E203` exception for flake8...

It seems black still insists on the space, and flake8 does not like it.

Reply:
I'm really sorry! This is my mistake and after checking a bit online it seems to be a known problem where `flake8` is inconsistent with pep8 on list slicing. Let us just let black deal with whitespace formatting like they recommend: https://black.readthedocs.io/en/stable/compatible_configs.html#flake8